docs/user/canvas.asciidoc

```diff
@@ -5,6 +5,8 @@
 [partintro]
 --
 
+*Note:* Canvas is only available for upgraded installations with existing workpads.
```

Comment:
Do we plan to deprecate Canvas in 9.0 as part of this PR? If so, maybe we should have something like:

```
deprecated::[9.0.0,Canvas has been deprecated and will be removed in 10.0.0]
```

If we plan to deprecate Canvas in our docs in a different PR, that's okay with me as long as both PRs target 9.0. Otherwise, I think users will be confused by this note.

Reply:
The plan is to add a deprecation notice to the docs. I did not want to bog this PR down with those details. Opened https://github.com/elastic/kibana/issues/200649 to track